### PR TITLE
Added class path entries

### PR DESCRIPTION
**Pull request by VinoshanT**
Signed-off-by: Vinoshan Tharmalenkam <Vinoshan.Tharmalenkam@ibm.com>

### CODE REVIEW

File: runtime/util/cphelp.c [hunk 1]
```diff
@@ -207,7 +210,12 @@ addJarToSystemClassLoaderClassPathEntries(J9JavaVM *vm, const char *filename)
 		classPathLength += oldEntries[entryIndex].pathLength + 1;	/* add 1 for a null character */
 	}
 	/* Copy and grow the classPathEntries array */
-	newEntries = (J9ClassPathEntry*) j9mem_allocate_memory(sizeof(J9ClassPathEntry) * (entryCount + 1) + classPathLength, J9MEM_CATEGORY_CLASSES);
+	if(IS_COLD_RUN(vm)){
```
Comment (akshayben) [inline runtime/util/cphelp.c]: nit: space after if

File: runtime/util/cphelp.c [hunk 2]
```diff
@@ -244,12 +252,21 @@ addJarToSystemClassLoaderClassPathEntries(J9JavaVM *vm, const char *filename)
 		newCount = entryCount + 1;
 		classLoader->classPathEntries = newEntries;
 		classLoader->classPathEntryCount = newCount;
-		j9mem_free_memory(oldEntries);
+		if(IS_COLD_RUN(vm)){
```
Comment (akshayben) [inline runtime/util/cphelp.c]: nit: space after if and before parenthesis

File: runtime/util/cphelp.c [hunk 3]
```diff
@@ -244,12 +252,21 @@ addJarToSystemClassLoaderClassPathEntries(J9JavaVM *vm, const char *filename)
...
+		} else {
+			j9mem_free_memory(oldEntries);
+		}
+		
 	}
 done:
 	/* If any error occurred, discard any allocated memory and throw OutOfMemoryError */
 	if (0 == newCount) {
-		j9mem_free_memory(newEntries);
+		if(IS_COLD_RUN(vm)){
```
Comment (akshayben) [inline runtime/util/cphelp.c]: nit: space after if and before parenthesis

**Review by akshayben — APPROVED** [review]

File: runtime/util/cphelp.c [hunk 4]
```diff
@@ -207,7 +210,12 @@ addJarToSystemClassLoaderClassPathEntries(J9JavaVM *vm, const char *filename)
 		classPathLength += oldEntries[entryIndex].pathLength + 1;	/* add 1 for a null character */
 	}
 	/* Copy and grow the classPathEntries array */
-	newEntries = (J9ClassPathEntry*) j9mem_allocate_memory(sizeof(J9ClassPathEntry) * (entryCount + 1) + classPathLength, J9MEM_CATEGORY_CLASSES);
+	if(IS_COLD_RUN(vm)){
+		newEntries = (J9ClassPathEntry*) imem_allocate_memory(sizeof(J9ClassPathEntry) * (entryCount + 1) + classPathLength, J9MEM_CATEGORY_CLASSES);
```
Comment (Ershany-IBM) [inline runtime/util/cphelp.c:214]: Minor nitpick, but we have been doing (J9ClassPathEntry *)

File: runtime/util/cphelp.c [hunk 5]
```diff
@@ -207,7 +210,12 @@ addJarToSystemClassLoaderClassPathEntries(J9JavaVM *vm, const char *filename)
...
+	} else {
+		newEntries = (J9ClassPathEntry*) j9mem_allocate_memory(sizeof(J9ClassPathEntry) * (entryCount + 1) + classPathLength, J9MEM_CATEGORY_CLASSES);
```
Comment (Ershany-IBM) [inline runtime/util/cphelp.c:216]: Same as comment above

**Review by Ershany-IBM — APPROVED** [review]

**Pull request merged**